main.js

```diff
@@ -0,0 +1,12 @@
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
We should not validate the data by default, this should be validate when --

1. User submitted the form
2. User updated the form field